### PR TITLE
Serenity updates - October 2020 - Felix news

### PR DESCRIPTION
**Pull request by NolwennP**
Left to do:
- [x] Add screenshots
- [x] Add doc links
- [x] Add "More" links

### CODE REVIEW

File: content/updates/2020-10/1-Stop-your-imports-and-exports.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 - CE
 ---
 
 # Stop your imports and exports
-::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/TODO.html"
 :::
 
-TODO
+A brand new `Stop` button is now displayed in your process tracker. It allows you to stop Excel and CSV imports/exports you launched. Super helpful whenever you launch a job by mistake and don't want to wait for it to execute until the end. 😉
```
Comment (LaureBro) [inline content/updates/2020-10/1-Stop-your-imports-and-exports.md:12]:
```suggestion
A brand new `Stop` button is now displayed in your process tracker. It enables you to stop the Excel and CSV imports/exports you launched. Super helpful whenever you launch a job by mistake and don't want to wait for it to execute until the end. 😉
```

File: content/updates/2020-10/1-Stop-your-imports-and-exports.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+
+![The new `Stop` button in the process tracker](../img/TODO.png)
+
+A new `Allowed to stop a job` permission allows you to fine-tune which user roles will be able to see this new action.
```
Comment (LaureBro) [inline content/updates/2020-10/1-Stop-your-imports-and-exports.md:16]:
```suggestion
This button comes with a dedicated permission `Allowed to stop a job`. If you need it, ask your PIM admin to give you the right to do so.

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
+pim_announcement_alt_img: "TODO"
 pim_announcement_audience:
 - EE
 - CE
 ---
 
 # Jobs and process tracker improvements
 
-TODO
+Follow the estimated time left for your ongoing jobs, quickly jump to the process tracker once you launched a quick export, and enjoy some enhancements directly inside your process tracker grid.
```
Comment (LaureBro) [inline content/updates/2020-10/6-Job-and-process-tracker-improvements.md:10]:
```suggestion
We made several enhancements to our process tracker: we now display a progress bar with the estimated remaing time before your job is completed, a shortcut to the process tracker when you launch a quick export and we added a new filter in the process tracker grid.
```

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md [hunk 4]
```diff
@@ -1,22 +1,40 @@
...
 
 ## Progress bar and time left for your ongoing jobs
-::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/TODO.html"
 :::
 
+In the process tracker, in the detail of each ongoing job, we now compute an estimation of the time left for each job step. A progress bar is also displayed.
```
Comment (LaureBro) [inline content/updates/2020-10/6-Job-and-process-tracker-improvements.md:16]:
```suggestion
From now on, you will be able to see an estimation of the remaining time and a progress bar for all your ongoing jobs. 
```
Do you know if this applies to all jobs? imports and exports?

**Review by LaureBro — CHANGES_REQUESTED** [review]
la news à Félix comments

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md (hunk 3 above)
Comment (NolwennP) [reply content/updates/2020-10/6-Job-and-process-tracker-improvements.md:10]: Sentence construction is weird at the end. You have 3 things but only 2 verbs. And there is a typo at "remaining". ;)

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md (hunk 4 above)
Comment (NolwennP) [reply content/updates/2020-10/6-Job-and-process-tracker-improvements.md:16]: Import and export. @felix-couprie Can you confirm?

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md (hunk 3 above)
Comment (LaureBro) [reply content/updates/2020-10/6-Job-and-process-tracker-improvements.md:10]: I've rearranged the text and fixed typos :)
```suggestion
Check our latest process tracker enhancements: a progress bar and the estimated remaining time for your ongoing jobs, a shortcut for a rapid access to quick exports job details, and a new sorting option in the process tracker grid.
```

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md (hunk 4 above)
Comment (felix-couprie) [reply content/updates/2020-10/6-Job-and-process-tracker-improvements.md:16]: Hello Nolwenn and Laure!
Yes, it applies to import and export jobs that are listed in the Notion page. Note that we way not be able to have the progress for every step when tracking a custom job.

Comment (NolwennP) [reply content/updates/2020-10/6-Job-and-process-tracker-improvements.md:16]: No future in sentences, @LaureBro. I suggest:
```suggestion
You are now able to see an estimation of the remaining time and a progress bar for all your ongoing jobs. 
```

**Comment by felix-couprie** [comment]
Hello Nolwenn & Laure,
PR is OK for me.

- Here's the link for the quick export documentation: https://help.akeneo.com/pim/serenity/articles/quick-export.html#how
- And here's the link for the process tracker improvements on filters: https://help.akeneo.com/pim/serenity/articles/monitor-jobs.html#searching-for-a-job

Thanks a lot!

**Review by felix-couprie — APPROVED** [review]
Hello, 
Thanks a lot, PR approved.

- Here's the link for the quick export documentation: https://help.akeneo.com/pim/serenity/articles/quick-export.html#how
- And here's the link for the process tracker improvements on filters: https://help.akeneo.com/pim/serenity/articles/monitor-jobs.html#searching-for-a-job

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md [hunk 5]
```diff
@@ -1,22 +1,36 @@
...
-TODO
-
-## Progress bar and time left for your ongoing jobs
-::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/.html"
-:::
+Check our latest process tracker enhancements: a shortcut for a rapid access to quick exports job details, and a new sorting option in the process tracker grid.
 
 ## Easy access to the detail of your quick export once launched
-::: meta-data type="Improvement" features="Productivity" available="mid-October" link-to-doc="../articles/.html"
+::: meta-data type="Improvement" features="Productivity" available="mid-October" link-to-doc="../articles/TODO.html"
```
Comment (NolwennP) [inline content/updates/2020-10/6-Job-and-process-tracker-improvements.md:14]:
```suggestion
::: meta-data type="Improvement" features="Productivity" available="mid-October" link-to-doc="../articles/quick-export.html#how"
```

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md [hunk 6]
```diff
@@ -1,22 +1,36 @@
...
 :::
 
+When you launch a quick product export from the product grid, you can now easily jump to this task's progress detail in the process tracker. Indeed, you'll find a link to the job detail directly inside the flash message that confirms the quick export launch.
+
+![Flash message with the direct link to the quick export detail](../img/flash-message-with-direct-link-to-the-quick-export-detail.png)
+
 ## Process tracker display enhancements
-::: meta-data type="New" features="Productivity" available="mid-October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="mid-October" link-to-doc="../articles/TODO.html"
```
Comment (NolwennP) [inline content/updates/2020-10/6-Job-and-process-tracker-improvements.md:22]:
```suggestion
::: meta-data type="New" features="Productivity" available="mid-October" link-to-doc="../articles/monitor-jobs.html#searching-for-a-job"
```

**Review by LaureBro — APPROVED** [review]

**Pull request merged**